docs/source/build-with-bentoml/gpu-inference.rst

```diff
@@ -31,14 +39,23 @@ In systems with multiple GPUs, each GPU is assigned an index starting from 0 (``
 
 To use a specific GPU:
 
-.. code-block::
+.. code-block:: python
+
+    import bentoml
 
     @bentoml.service(resources={"gpu": 2})
     class MultiGPUService:
+        # Load Hugging Face models
+        model1_path = bentoml.models.HuggingFaceModel("org_name/model1_id")
+        model2_path = bentoml.models.HuggingFaceModel("org_name/model2_id")
+
+        # Use a model from the Model Store or BentoCloud
+        # model_path = bentoml.models.BentoModel("model_name:latest")
+
         def __init__(self):
             import torch
-            self.model1 = torch.load('model1.pth').to("cuda:0")  # Using the first GPU
-            self.model2 = torch.load('model2.pth').to("cuda:1")  # Using the second GPU
+            self.model1 = torch.load(self.model1_path).to("cuda:0")  # Using the first GPU
+            self.model2 = torch.load(self.model2_path).to("cuda:1")  # Using the second GPU
```

Comment:
I don't think this is correct, because `torch.load` requires the actual `.pt` files, and model1_path only returns the path to the model cache. So probably needed to include a `os.path.join(self.model1_path, "weight.pt")`